$MirOS: ports/www/lynx/patches/patch-src_LYMain_c,v 1.12 2010/12/23 20:34:02 tg Exp $

	site policy
	also contains spelling fixes; suggested for upstream inclusion

--- src/LYMain.c.orig	Mon Feb 20 02:13:33 2012
+++ src/LYMain.c	Sat Mar  3 18:20:22 2012
@@ -1199,6 +1199,9 @@ int main(int argc,
     StrAllocCopy(LYUserAgent, LYNX_NAME);
     StrAllocCat(LYUserAgent, "/");
     StrAllocCat(LYUserAgent, LYNX_VERSION);
+#ifdef __MirBSD__
+    StrAllocCat(LYUserAgent, " (compatible; MirBSD; UNIX)");
+#endif
     if (HTLibraryVersion) {
 	StrAllocCat(LYUserAgent, " libwww-FM/");
 	StrAllocCat(LYUserAgent, HTLibraryVersion);
@@ -1666,7 +1669,7 @@ int main(int argc,
      * If the input is not a tty, we are either running in cron, or are
      * getting input via a pipe:
      *
-     * a) in cron, none of stdin/stdout/stderr are tty's.
+     * a) in cron, none of stdin/stdout/stderr are ttys.
      * b) from a pipe, we should have either "-" or "-stdin" options.
      */
     if (!LYGetStdinArgs
@@ -2131,10 +2134,7 @@ int main(int argc,
      * synchronized.  - FM
      */
     if (isEmpty(bookmark_page)) {
-	temp = NULL;
-	HTSprintf0(&temp, "lynx_bookmarks%s", HTML_SUFFIX);
-	set_default_bookmark_page(temp);
-	FREE(temp);
+	set_default_bookmark_page("./.etc/bookmark.htm");
     }
     if (isEmpty(BookmarkPage)) {
 	set_default_bookmark_page(bookmark_page);
@@ -3027,7 +3027,7 @@ within userdefs.h." },
 #endif
 	{ "file_url", "\
 disallow using G)oto, served links or bookmarks for\n\
-file: URL's" },
+file: URLs" },
 	{ "goto", "disable the 'g' (goto) command" },
 #if !defined(HAVE_UTMP) || defined(VMS) /* not selective */
 	{ "inside_ftp", "\
@@ -3065,7 +3065,7 @@ disallow execution of Lynx CGI URLs" },
 	{ "mail", "disallow mail" },
 	{ "multibook", "disallow multiple bookmark files" },
 	{ "news_post", "disallow USENET News posting." },
-	{ "option_save", "disallow saving options in .lynxrc" },
+	{ "option_save", "disallow saving options in .etc/lynxrc" },
 #if !defined(HAVE_UTMP) || defined(VMS) /* not selective */
 	{ "outside_ftp", "\
 disallow ftps coming from outside your\n\
@@ -3093,9 +3093,9 @@ disallow telnets coming from outside you
 	{ "print", "disallow most print options" },
 	{ "shell", "\
 disallow shell escapes, and lynxexec, lynxprog or lynxcgi\n\
-G)oto's" },
+G)oto" },
 	{ "suspend", "disallow Control-Z suspends with escape to shell" },
-	{ "telnet_port", "disallow specifying a port in telnet G)oto's" },
+	{ "telnet_port", "disallow specifying a port in telnet G)oto" },
 	{ "useragent", "disallow modifications of the User-Agent header" },
     };
     /* *INDENT-ON* */
@@ -3231,38 +3231,7 @@ static int version_fun(char *next_arg GC
     printf("%s\n", result);
     free(result);
 
-#ifndef __DATE__
-#define __DATE__ ""
-#endif
-#ifndef __TIME__
-#define __TIME__ ""
-#endif
-
-/*
- * SYSTEM_NAME is set by the configure script.  Show build date/time for other
- * systems, according to predefined compiler symbols.
- */
-#ifdef SYSTEM_NAME
-    printf(gettext("Built on %s %s %s\n"), SYSTEM_NAME, __DATE__, __TIME__);
-#else
-#ifdef __CYGWIN__
-    printf("Compiled by CYGWIN (%s %s).\n", __DATE__, __TIME__);
-#else
-#ifdef __BORLANDC__
-    printf("Compiled by Borland C++ (%s %s).\n", __DATE__, __TIME__);
-#else
-#ifdef _MSC_VER
-    printf("Compiled by Microsoft Visual C++ (%s %s).\n", __DATE__, __TIME__);
-#else
-#ifdef __DJGPP__
-    printf("Compiled by DJGPP (%s %s).\n", __DATE__, __TIME__);
-#else
-    printf("Compiled at (%s %s).\n", __DATE__, __TIME__);
-#endif /* __DJGPP__ */
-#endif /* _MSC_VER */
-#endif /* __BORLANDC__ */
-#endif /* __CYGWIN__ */
-#endif
+    printf("No compilation information available.\n");
 
     puts("");
     puts(gettext("Copyrights held by the Lynx Developers Group,"));
